test/new-e2e/tests/otel/otel-agent/complete_test.go

```diff
@@ -43,7 +44,9 @@ agents:
           value: 'disable_receive_resource_spans_v2,disable_operation_and_resource_name_logic_v2'
 `
 	t.Parallel()
-	e2e.Run(t, &completeTestSuite{}, e2e.WithProvisioner(awskubernetes.KindProvisioner(awskubernetes.WithAgentOptions(kubernetesagentparams.WithHelmValues(values), kubernetesagentparams.WithOTelAgent(), kubernetesagentparams.WithOTelConfig(completeConfig)))))
+	e2e.Run(t, &completeTestSuite{}, e2e.WithProvisioner(
+		provkindvm.Provisioner(provkindvm.WithRunOptions(scenkindvm.WithAgentOptions(kubernetesagentparams.WithHelmValues(values), kubernetesagentparams.WithOTelAgent(), kubernetesagentparams.WithOTelConfig(completeConfig))))),
+	)
```

Comment:
nit: align style with `test/new-e2e/tests/otel/otel-agent/host_metrics_receiver_test.go`
```suggestion
	e2e.Run(t, &completeTestSuite{},
		e2e.WithProvisioner(
			provkindvm.Provisioner(
				provkindvm.WithRunOptions(
					scenkindvm.WithAgentOptions(
						kubernetesagentparams.WithHelmValues(values),
						kubernetesagentparams.WithOTelAgent(),
						kubernetesagentparams.WithOTelConfig(completeConfig),
					)
				)
			)
		)
	)
```